decentraland/explorer
# feat: explore -part3- hud

TEST-LINK: https://play.decentraland.zone/branch/feat/explore-part3-hud-controller/index.html?ENV=org&ENABLE_EXPLORE_HUD

**How to test**

- Explore button is located under the mini map. click to open and close the menu
- Use X key to open close Explore Menu
- Press C to open controls and check the X key reference is included
- Mouse hover UI buttons (chat, friends, PMs, explore) shows labels

- Explore HUD shows all the POIs (sorted by most crowded to less). Thumbnails only visible on those that has one submited.
- Online players visible
- Mouse hover shows jump in button
- Click on info button opens scene detail plus Jump in button too 
- Magic button takes players to a new scene they havent visited yet
